.github/CODESTYLE.md

```diff
@@ -0,0 +1,305 @@
+# WRF-Hydro Fortran Conventions Guide
+
+# Fortran standard
+New code contributions should be written to at least Fortran 2003 standard.
```

Comment:
Link to: https://gcc.gnu.org/wiki/GFortranStandards#Fortran_2003 or http://fortranwiki.org/fortran/show/Fortran+2003 or not worth it?

Reply:
added link